bin/start-systemtest

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+
+##
+
+## checking args
+if [ $# -ne 3 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD DISPATCHER_ADDRESS"
+  exit 1
+fi
+
+# args
+readonly USER_EMAIL=${1}
+readonly USER_PASSWORD=${2}
+readonly DISPATCHER_ADDRESS=${3}
+
+readonly SANDBOX=$(cd "$(dirname "${0}")" && pwd)
```

Comment:
isn't it the same of $(pwd)?

Reply:
I imagine that if the user runs the script from a folder beyond which the 'start-systemtest' script is contained, it may be that the other scripts are not found. Does your solution cover this? If so I can perform the test using it to ensure this

Reply:
It's ok for me